src/app/admin/components/hierarchy-builder/hierarchy-builder.component.ts

```diff
@@ -0,0 +1,117 @@
+
+import { NestedTreeControl} from '@angular/cdk/tree';
+import { Component, Input, OnInit, Output, EventEmitter, ChangeDetectorRef } from '@angular/core';
+import { LearningObject } from '@entity';
+import { HierarchyService } from 'app/admin/core/hierarchy.service';
+import { ToastrOvenService } from 'app/shared/modules/toaster/notification.service';
+import { LearningObjectNode, TreeDataSource } from './tree-datasource';
+
+@Component({
+  selector: 'clark-hierarchy-builder',
+  templateUrl: 'hierarchy-builder.component.html',
+  styleUrls: ['hierarchy-builder.component.scss'],
+})
+export class HierarchyBuilderComponent implements OnInit {
+
+  // This is just dummy data that is replaced in the OnInit function with the parent data
+  TREE_DATA: LearningObjectNode[] = [
+    {
+      "name": "paige",
```

Comment:
```suggestion
      "name": "paige the rage",
```
Gotta leave your mark on the team